src/tentacles/resources/mlflow_session.py

```diff
@@ -68,14 +77,30 @@ def _get_run_name_from_context(
         str
             Run name
         """
-        asset_key = get_asset_key(context)
-        dagster_run_id = get_run_id(context, short=True)
 
-        run_name = f"{asset_key}-{dagster_run_id}"
+        if run_name_prefix is None:
```

Comment:
Should we just call the param `prefix`? The method name `_get_run_name_from_context` makes it clear that the prefix concerns a run name